active/0009-import-export-feature/README.md

```diff
@@ -0,0 +1,340 @@
+- Start Date: 2024-11-12
+- RFC PR: https://github.com/datahub-project/rfcs/pull/9
+- Implementation PR(s): 
+
+# Import/Export Feature
+
+## Summary
+
+This feature will add the ability to both export datasets to CSV files and import them back into DataHub from those CSV files, using the UI. Code is already implemented for this feature, though further work may need to be done. This RFC details the Implementation in its current state.
+
+## Motivation
+
+This feature was developed with the intention to mimic the import/export functionality present in Collibra. It can be used for moving datasets between instances of DataHub, which may be useful for enterprise-level users. Though it is not a strictly necessary feature, the DataHub team has expressed interest in adding it to the DataHub project.
+
+## Requirements
+
+This feature as it is currently implemented is only intended to support:
+- Export to CSV of individual datasets.
+- Export to CSV of all datasets within a container.
+- Import from CSV of previously exported data.
+
+## Non-Requirements
+
+This feature is not intended to add a REST API for import/export like that of Collibra. It is only intended for use through the UI.
```

Comment:
Maybe this is where we declare that we are not intending to support a from-scratch import of csv metadata which has not been sourced from DataHub